site/docs/es/plugins/files.md

```diff
@@ -0,0 +1,153 @@
+# Manejo de archivos simplificado en grammY (`files`)
+
+Este plugin te permite descargar fácilmente archivos de los servidores de Telegram, y obtener una URL para que puedas descargar el archivo tú mismo.
+
+## Descargando Archivos
+
+Necesitas pasar el token de tu bot a este plugin porque debe autenticarse como tu bot cuando descarga archivos.
+Este plugin entonces instala el método `download` en los resultados de la llamada `getFile`.
+Ejemplo:
+
+<CodeGroup>
+  <CodeGroupItem title="TypeScript" active>
+
+```ts
+import { Bot, Context } from "grammy";
+import { FileFlavor, hydrateFiles } from "@grammyjs/files";
+
+// Transformative Context flavor
+type MyContext = FileFlavor<Context>;
+
+// Crea un bot.
+const bot = new Bot<MyContext>("");
+
+// Usa el plugin.
+bot.api.config.use(hydrateFiles(bot.token));
+
+// Descarga vídeos y GIFs en ubicaciones temporales.
+bot.on([":video", ":animation"], async (ctx) => {
+  // Prepare el archivo para su descarga.
+  const file = await ctx.getFile();
+  // Descargue el archivo en una ubicación temporal.
+  const path = await file.download();
+  // Imprime la ruta del archivo.
+  console.log("Archivo guardado en", path);
+});
+```
+
+</CodeGroupItem>
+ <CodeGroupItem title="JavaScript">
+
+```js
+import { Bot } from "grammy";
+import { hydrateFiles } from "@grammyjs/files";
+
+// Crea un bot.
+const bot = new Bot("");
+
+// Usa el plugin.
+bot.api.config.use(hydrateFiles(bot.token));
+
+// Descarga vídeos y GIFs en ubicaciones temporales.
+bot.on([":video", ":animation"], async (ctx) => {
+  // Prepare el archivo para su descarga.
+  const file = await ctx.getFile();
+  // Descargue el archivo en una ubicación temporal.
+  const path = await file.download();
+  // Imprime la ruta del archivo.
+  console.log("Archivo guardado en", path);
+});
+```
+
+</CodeGroupItem>
+ <CodeGroupItem title="Deno">
+
+```ts
+import { Bot, Context } from "https://deno.land/x/grammy/mod.ts";
+import {
+  FileFlavor,
+  hydrateFiles,
+} from "https://deno.land/x/grammy_files/mod.ts";
+
+// Transformative Context flavor
+type MyContext = FileFlavor<Context>;
+
+// Create a bot.
```

Comment:
Wink

Reply:
Thanks. `Transformative Context flavor` will remain, I will change only the second.

Reply:
Thanks!